lib/packages/core/src/api/__tests__/e2e/messageApi.spec.e2e.ts

```diff
@@ -6,7 +6,7 @@ import {sendEncryptedTextMessage} from '../../factories/message/sendEncryptedTex
 import {getTransaction} from '../../factories/transaction/getTransaction';
 import {isAttachmentVersion} from '../../../attachment';
 import {sendMessage} from '../../factories/message';
-import {FeeQuantPlanck} from '../../../constants';
+import {FeeQuantPlanck} from '@burstjs/util';
```

Comment:
This is probably not right - should we be using aliases for things inside `lib`?

Reply:
humm... we always did it like that. But actually using other packages requires at least a re-tsc (or even bootstrap)